doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -19,23 +22,66 @@ version to install - when not specified, the latest version will be installed in
 
 ## Upgrading the Operator
 
-To upgrade the Operator, use the dedicated `upgrade-operator` command.
+The procedure for upgrading the Operator depends on the version of the Operator you want to
+upgrade from.
+
+### Upgrading the Operator (>v1.0)
+
+Astarte Operator's Helm chart is available since v1.0 and therefore the current upgrade procedure applies only
+if you are dealing with at least v1.0.
+
+To upgrade the Operator, use the dedicated `helm upgrade` command:
 
 ```bash
-astartectl cluster upgrade-operator
+helm upgrade astarte-operator astarte/astarte-operator [--version <new-version>]
 ```
 
-Just like the `install-operator` command, the `--version` switch allows to specify the version to upgrade to -
-when not specified, the latest version will be fetched and used.
+The optional `--version` switch allows to specify the version to upgrade to - when not specified, the latest
+version will be fetched and used.
+
+Currently, only the first version of the chart is available within `helm.astarte-platform.org` and the `helm upgrade`
+operation will lead to an error. Bear with that as long as new versions will be released.
```

Comment:
`Bear with that until the next version gets released`

Also, does upgrade actually fails if there's only a single version? If so, we have to remember to remove this as soon as the next version is released.

Reply:
Should we rephrase it as to make it more general and to account for further scenarios?

Reply:
Imho the point is discovering what happens if you upgrade and:
- you already are at the latest version
- there are no other versions (which should actually be a subset of the previous case)
If helm just prints something like "You already are at the latest available version" or something similar, I think we can remove this paragraph altogether